src/Client/Components/ExpansionList/AppExpansionList.razor.cs

```diff
@@ -20,4 +20,23 @@ public partial class AppExpansionList
 
     [Parameter]
     public EventCallback<InterviewQuestion> OnOpenResetDialog { get; set; }
+
+    [Parameter]
+    public EventCallback OnRatingChanged { get; set; }
+
+    [Parameter]
+    public bool IsSkeleton { get; set; }
+
+    public void RatingChanged()
+        => OnRatingChanged.InvokeAsync();
+
+    public IEnumerable<InterviewQuestion> Search(string value)
+    {
+        if (InterviewQuestions is null)
+        {
+            throw new InvalidOperationException(nameof(InterviewQuestions));
+        }
+
+        return InterviewQuestions.Where(x => x.Contains(value));
```

Comment:
It is slightly better practice to name your query variables according to the contents of the enumeration. It's just a bit easier to read.

```suggestion
        return InterviewQuestions.Where(q => q.Contains(value));
```